src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -96,7 +107,10 @@ async function makeLambdaDebugFile(params: {
     }
     const logger = getLogger()
 
-    const [handlerFilePrefix, handlerFunctionName] = params.handlerName.split('.')
+    const splitHandlerName = params.handlerName.split('.')
+    const handlerFunctionName = splitHandlerName.pop()
```

Comment:
This is a little hard to read because you're mutating the splitHandlerName with `pop`. Consider using slice to keep the splitHandlerName intact